src/style.css

```diff
@@ -2,0 +1,124 @@
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  font-style: normal;
+  src: url(fonts/Avenir-Book.ttf) format('truetype');
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 600;
+  font-style: normal;
+  src: url(fonts/Avenir-Heavy.ttf) format('truetype');
+}
+
+.page__body {
+  font-family: Avenir, 'Helvetica Neue', Arial, sans-serif;
+  font-size: 16px;
+}
+
+.search-bar {
+  display: flex;
+  justify-content: center;
+  gap: 20px;
+  margin-top: 20px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.search-bar-big {
+  width: 100%;
+  min-height: 70px;
+}
+
+.search-bar-small {
+  width: 100%;
+  min-height: 42px;
+}
+
+.search-bar__label {
+  position: relative;
+  display: block;
+  width: 100%;
+}
+
+.search-bar__label::after {
+  position: absolute;
+  content: '';
+  display: block;
+}
+
+.search-bar__label-big::after {
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.search-bar__label-small::after {
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.search-bar__input {
+  width: 100%;
+  height: 100%;
+  border: 1px solid #e1e7ed;
+  font-family: inherit;
+  border-radius: 4px;
+  font-weight: 300;
+  color: #3d4e61;
+  box-shadow: 0 1px 8px 0 #3d4e6133;
+  outline: none;
+}
+
+.search-bar__input:hover {
+  box-shadow: 0 3px 8px 0 #3d4e6133;
+}
+
+.search-bar__input:focus {
+  border: 1px solid #e1e7ed;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  font-weight: 600;
+  color: #3d4e61;
+}
+
+.search-bar__input-big {
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22;
```

Comment:
The 'line-height' property should have a unit specified, such as 'px'. Without a unit, it may not render as expected.